spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationBuilder.java

```diff
@@ -35,21 +38,25 @@
 import org.springframework.boot.bind.RelaxedNames;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.binding.BindableProxyFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.core.env.PropertySources;
 import org.springframework.integration.monitor.IntegrationMBeanExporter;
 import org.springframework.util.StringUtils;
 
 /**
- * Application builder for {@link AggregateApplication}.
+ * Application builder for {@link AggregateApplicationUtils}.
  *
  * @author Dave Syer
  * @author Ilayaperumal Gopinathan
  * @author Marius Bogoevici
  * @author Venil Noronha
  */
-public class AggregateApplicationBuilder {
+@EnableBinding
```

Comment:
yeah, I missed that. It is just that we set the `parentSources` with the `ParentConfiguration` but that is not used at all in case of `bean` type obviously :-)